### PR TITLE
Adds OracleServicer GPRC class and implements GetSpace, UpdateSpace, and CreateTrial

### PR DESCRIPTION
**Pull request by omalleyt12**
Most of the LOC in this CL is from the generated pb2 files

Will implement the other OracleServicer methods in a follow-up PR, and then create an OracleClient

### CODE REVIEW

File: kerastuner/distribute/oracle_servicer.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import grpc
+import os
+import time
+
+from ..engine import hyperparameters as hp_module
+from ..protos import service_pb2
+from ..protos import service_pb2_grpc
+
+
+class OracleServicer(service_pb2_grpc.OracleServicer):
```
Comment (fchollet) [inline kerastuner/distribute/oracle_servicer.py]: What was the motivation for the name "OracleServicer"? What would be some other options?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,3 +54,15 @@ information on using pull requests.
 
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google.com/conduct/).
+
+
+# Rebuilding Protos
```
[the remark this answers is not in the document]
Comment (omalleyt12) [reply CONTRIBUTING.md]: Done

File: kerastuner/distribute/oracle_servicer.py (hunk 1 above)
Comment (omalleyt12) [reply kerastuner/distribute/oracle_servicer.py]: Done, went with oracle_chief

**Comment by omalleyt12** [comment]
Thanks for the review!

**Pull request merged**